### ISSUE TITLE
Runtime error in training

### ISSUE THREAD

**Issue by sanchitvohra** [Lornatang/SRGAN-PyTorch#37]
![Screenshot from 2021-08-19 00-47-36](https://user-images.githubusercontent.com/23478787/130014817-ca35dd58-70f7-4628-91b2-07aab68079bc.png)

**Comment by sanchitvohra** [comment]
You're disabling gradients while computing super resolution image forward pass through generator, hence the generator loss has no gradients and causes runtime error on backprop

**Comment by Lornatang** [comment]
I fix.

**Comment by Lornatang** [comment]
The new version weight has been uploaded, and some known bugs have been fixed
